frontend/src/pages/home.tsx

```diff
@@ -10,39 +10,60 @@ const Home = () => {
+		buttonLabel: "Learn more...",
 		buttonUrl: "/loremipsum"
 	};
 
 	const subFeaturesProps = [
 		{
-			headline: "Lorem Ipsum 1",
+			headline: "Tutorial Expeditions",
 			content:
-				"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
-			linkLabel: "Lorem Ipsum 1 >",
+				"Every voyage begins with a tutorial expedition. Prove your skills as a 'Captain' or showcase your dedication as a 'Cadet' by completing these starter assignments. Get acquainted with our platform and learn the ropes of MentorShip.",
+			linkLabel: "Learn more... >",
 			url: "/loremipsum1"
 		},
 		{
-			headline: "Lorem Ipsum 2",
+			headline: "Craft Your Ship",
 			content:
-				"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
-			linkLabel: "Lorem Ipsum 2 >",
+				"As a 'Captain,' take charge by creating a 'Ship' and crafting your unique 'Mission.' Design the challenges, set the deadlines, and guide your 'Crew' through uncharted coding waters towards your goal.",
+			linkLabel: "Learn more... >",
 			url: "/loremipsum2"
 		},
 		{
-			headline: "Lorem Ipsum 3",
+			headline: "Assemble or Join a Crew",
 			content:
-				"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
-			linkLabel: "Lorem Ipsum 3 >",
+				"Captains can browse a pool of motivated 'Cadets' who have completed the tutorials. Choose your crew wisely and embark on your voyage. On the flip side, 'Cadets' can explore 'Ships' and join projects that resonate with their aspirations.",
+			linkLabel: "Learn more... >",
+			url: "/loremipsum3"
+		},		
+		{
+			headline: "Set Sail Together",
+			content:
+				"The voyage begins! Collaborate with your 'Crew' to tackle tasks and challenges that mimic the real software development world. Peer code reviews, pull requests, and teamwork become second nature as you navigate your 'Mission.'",
+			linkLabel: "Learn more...>",
+			url: "/loremipsum3"
+		},
+		{
+			headline: "Advancement and Rank",
+			content:
+				"As you navigate the development seas, gain experience and rise through the ranks. Both 'Captains' and 'Cadets' can unlock new features, making the journey even more rewarding.",
+			linkLabel: "Learn more... >",
+			url: "/loremipsum3"
+		},
+		{
+			headline: "Community and Support",
+			content:
+				"MentorShip is more than a platform—it's a community of passionate developers, united by a love for coding and learning. Forge lasting connections, share insights, and contribute to the growth of fellow developers.",
+			linkLabel: "Learn more... >",
```

Comment:
Lets create a folder in pages for `home` and add the file to it, then create a new file called `constants.ts` and export all of these into a file. Once done rename all the variables to UPPERCASE to signify that they are unchanging variables (constants)